tests/shared/src/test/scala/scala/meta/tests/parsers/ScaladocParserSuite.scala

```diff
@@ -144,10 +144,10 @@ class ScaladocParserSuite extends FunSuite {
           * $descriptionBody [[ $descriptionBody ]]
           * $descriptionBody
           *
-          * [[ $descriptionBody ]]!$descriptionBody
-          *
           * $descriptionBody
           * [[ $descriptionBody ]] $descriptionBody
+          * 
+          * [[ $descriptionBody ]]'s
```

Comment:
why reorder and not put this on original line 147? would have fewer changes

Reply:
Yep, but the original input from 147 was a little bit confusing: 
  `[[ Description  Body ]]!Description Body`  -> `Link("Description", Seq("Body"), "!Description")`